### PR TITLE
CDS-1 Created base landing page and updated fonts

### PR DESCRIPTION
**Pull request by Liannus**
Changelog:
- Updated fonts to have global.scss file for animations, etc
- Added mixins and mediaQueries files for dynamic fonts and dynamic sizing matching tailwind queries
- Added basic landing page to be iterated upon with Navbar

### CODE REVIEW

File: src/pages/LandingPage/LandingPage.scss [hunk 1]
```diff
@@ -1,14 +1,26 @@
+@import "styles/mixins.scss";
+@import "styles/mediaQueries.scss";
+
 .landing-page {
   height: 100vh;
   background-size: cover;
-  background-position: center;
+  background-position: right center;
```
Comment (forkball) [inline src/pages/LandingPage/LandingPage.scss]: Should we do this with tailwind?

https://tailwindcss.com/docs/background-position/

File: src/pages/LandingPage/LandingPage.scss [hunk 2]
```diff
@@ -1,14 +1,26 @@
+@import "styles/mixins.scss";
+@import "styles/mediaQueries.scss";
+
 .landing-page {
   height: 100vh;
   background-size: cover;
```
Comment (forkball) [inline src/pages/LandingPage/LandingPage.scss]: Should we do this with tailwind?

https://tailwindcss.com/docs/background-size/#app

**Review by forkball — COMMENTED** [review]
Looks good, just think you have some layout styles that should (?) be done with tailwind.

File: src/pages/LandingPage/LandingPage.scss (hunk 1 above)
Comment (Liannus) [reply src/pages/LandingPage/LandingPage.scss]: Yeah I can change this

File: src/pages/LandingPage/LandingPage.scss (hunk 2 above)
Comment (Liannus) [reply src/pages/LandingPage/LandingPage.scss]: background size, yes, media queries no.

We're going to need the media queries to match tailwind with scss.

Comment (forkball) [reply src/pages/LandingPage/LandingPage.scss]: Yeah, the comment was related to background size.

File: src/pages/LandingPage/LandingPage.scss (hunk 1 above)
Comment (Liannus) [reply src/pages/LandingPage/LandingPage.scss]: Done

File: src/pages/LandingPage/LandingPage.scss (hunk 2 above)
Comment (Liannus) [reply src/pages/LandingPage/LandingPage.scss]: Done

**Review by forkball — COMMENTED** [review]
Looks good now.

**Review by forkball — APPROVED** [review]

**Pull request merged**